.github/workflows/codespell.yml

```diff
@@ -0,0 +1,18 @@
+name: Codespell
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  spell-check:
+    if: github.event.pull_request.draft == false
```

Comment:
❤️ 
That's cool, although does mean someone couldn't find typos in their PR until it's open for review...

Reply:
I just copy pasted the full template from one of our other actions and changed the parts below the `if:`, wasn't really a considered choice! 🙂

I've also considered running it along side the [`smoke-tests`](https://github.com/pi-hole/pi-hole/blob/master/.github/workflows/test.yml#L19) here, and preventing `distro-test` from running until code-spell is satistfied.